core/src/property/backend/lww.rs

```diff
@@ -119,7 +111,9 @@ impl PropertyBackend for LWWBackend {
     }
 
     fn get_property_value_string(&self, property_name: &str) -> Option<String> {
-        self.values.read().unwrap().get(property_name).map(|v| String::from_utf8_lossy(v).to_string())
+        //let values = self.values.read().unwrap();
+        //values.get(property_name).map(|v| String::from_utf8_lossy(v).to_string())
+        unimplemented!()
```

Comment:
Yes, this can just cast everything to a string for now. Lets open a separate case for typed Predicate values